spec/koans.js

```diff
@@ -84,38 +85,44 @@ describe('`const` is like `let` plus read-only. ', () => {
 describe('`string.includes()` finds string within another string. ', () => {
 
   describe('find a single character', function() {
-    it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+    it('in a three char string', () => {
+      const searchString = ('xyz')
+      expect('xyz'.includes(searchString)).toBe(true);
     });
     it('reports false if character was not found', function() {
-      /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+      const expected = false
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
   describe('find a string', function() {
     it('that matches exactly', function() {
-      /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+      const findSome = (i) => 'xyz'.includes(i);
+
+      // function findSome (par) {
+      //   return "xyz".includes(par);
+      // }
+
+      expect(findSome('xyz')).toBe(true);
     });
   });
 
   describe('search for an empty string, is always true', function() {
     it('in an empty string', function() {
-      /* .... */
-      //expect(''.includes(x)).toBe(true);
+      const x = ''
+      expect(''.includes(x)).toBe(true);
     });
     it('in `abc`', function() {
-      /* .... */
-      //expect('abc'.includes(x)).toBe(true);
+      let x = 'abc';
+      expect('abc'.includes(x)).toBe(true);
```

Comment:
Yes, but you can also try with 
```js
let x = ''
```
This is about showing us one behaviour of the includes method : if you search for an empty string inside another string, it will always return true.